packages/experiments/src/components/Chiclet/ChicletXsmall.base.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { classNamesFunction } from '../../Utilities';
+import { IChicletCardStyles, IChicletCardStyleProps } from './ChicletCard.types';
+import { IChicletCardProps } from './ChicletCard.types';
+import { generateCommonHTML } from '../../utilities/chicletHelper';
+
+const getClassNames = classNamesFunction<IChicletCardStyleProps, IChicletCardStyles>();
+
+export class ChicletXsmallBase extends React.Component<IChicletCardProps, {}> {
+  private _classNames: { [key in keyof IChicletCardStyles]: string };
+
+  public render(): JSX.Element {
+    const { onClick, title, image, imageAlt, className, footer, theme, styles, url } = this.props;
+
+    const footerProvided = !!footer;
+    const imageProvided = !!image || !!imageAlt;
+
+    this._classNames = getClassNames(styles, { theme: theme!, className, footerProvided });
+
+    // if this element is actionable it should have an aria role
+    const role = onClick ? 'button' : 'link';
+    const tabIndex = onClick ? 0 : undefined;
```

Comment:
A consideration to explore in a future iteration (not necessarily this PR) is to have the root element be either `<a/>` or `<button/>` which would remove the need for the `role` and `tabIndex`.

The above would likely require modifying styles tho.